extensions/perchance-generator/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "perchance-generator",
+  "title": "Perchance Generator",
+  "description": "Input the project name and receive the generated text.",
```

Comment:
Project name?

Reply:
Yeah, I believe that's what they're called . . .

Reply:
Hmmm. I did a little more digging and found that they also called them _generator names_ OR project names, so will change to first one